pages/coinbase/signin.py

```diff
@@ -0,0 +1,28 @@
+from selenium import webdriver
+from selenium.webdriver.support import expected_conditions as EC  
+from selenium.webdriver.support.ui import WebDriverWait as wait
+from selenium.webdriver.common.by import By
+import secure.cookies
```

Comment:
Just import secure. You'll get all files in that directory (which is just these two).